portfolio.html

```diff
@@ -53,24 +54,32 @@ <h1>Not so interesting stories</h1>
       <div class="top-content">
         <img src="img/greatIdea.png" alt="greatIdea">
         <h2>A Great Idea</h2>
-        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <p>Iterative approaches to corporate strategy foster collaborative<br>thinking to further the overall value proposition.</p>
         <button onclick="window.location.href = 'https://github.com/Frankzhac/UI-III-Flexbox/tree/master/great-idea';">View code on GitHub</button>
       </div>
 
       <div class="top-content">
         <img src="img/sprintChallenge.png" alt="strategy">
         <h2>Sprint Challenge User Interface</h2>
-        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <p>Iterative approaches to corporate strategy foster collaborative<br>thinking to further the overall value proposition.</p>
         <button onclick="window.location.href = 'https://github.com/Frankzhac/Sprint-Challenge--User-Interface';">View code on GitHub</button>
```

Comment:
Whoa!! Adding some onClick! Very impressive. See if you can make this link redirect to a new tab.